snaps_boot/provision/hardware/pxe_utils.py

```diff
@@ -44,63 +44,73 @@ def __main(config, operation):
     bmc_dict = prov_dict.get('BMC')
     subnet_list = dhcp_dict.get('subnet')
     cpu_core_dict = prov_dict.get('CPUCORE')
-    pxe_server_configuration_listmap = tftp_dict.get('pxe_server_configuration')
-    buildPxeServer = None
-    ubuntuPxeServer = False
-    centosPxeServer = False
+    pxe_server_configuration_listmap = tftp_dict.get(
+        'pxe_server_configuration')
+    build_pxe_server = None
+    ubuntu_pxe_server = False
+    centos_pxe_server = False
     ubuntu_dict = {}
-    if pxe_server_configuration_listmap == None:
-       logger.error("Please define OS(ubuntu/centos) details in hosts.yaml to create pxe server")
-       exit(1)
+    if not pxe_server_configuration_listmap:
+        logger.error(
+            'Please define OS(ubuntu/centos) details in hosts.yaml to create '
+            'pxe server')
+        exit(1)
```

Comment:
If you leave this is it in not backwards compatible and it will have to be merged with issue-90 which does it differently.  I assume ubuntu and make a not of the deprecated host.yaml file.

Reply:
This block should function the same as the master version. We'll see what gets merged first :)